### DISCUSSION TITLE
HA und Systemvariablen vom Typ Zahl mit hahm Beschreibung wird nicht erkannt, wenn der Wert der Variable 0 in der Homematic ist.

### DISCUSSION THREAD

**Discussion by HannesOOE** [SukramJ/aiohomematic#1457] [Q&A]
Hallo alle zusammen!
Ich habe folgendes Problem:
Eine Homematic Variable vom Typ Zahl mit hahm in der Beschreibung wird im HA korrekt als number-Entity erkannt.
Hat die Variable irgend einen Wert <> 0, dann wird diese Variable richtig im HA dargestellt und kann auch geändert werden.
Ist der Wert der Variable in der Homematic jedoch 0, stellt HA den Wert als NaN -Wert bzw. Unbekannt dar und kann auch nicht bearbeitet werden.
Kennt wer dieses Problem?
Schon jetzt danke für eure Antwort!
LG Hannes

**Comment by SukramJ** [comment]
Das ist ein Bug. Wird in der nächsten Version behohen.

**Comment by HannesOOE** [comment]
Hallo SukramJ,Herzlichen Dank für die rasche Antwort und fürs Beheben!LG Hannes Am 10.03.2024 um 19:04 schrieb SukramJ ***@***.***>:﻿
Das ist ein Bug. Wird in der nächsten Version behohen.
